### PR TITLE
All home changes given the new execution functionality

### PR DESCRIPTION
**Pull request by matuella**

### CODE REVIEW

File: lib/application/constants/strings.dart [hunk 1]
```diff
@@ -16,18 +16,51 @@ const collectionsExploreTab = 'Explorar';
...
 const collectionsSectionHeaderSeeAll = 'Ver todos';
-const collectionsMemoryStability = 'Estabilidade da Memória';
-String collectionsCompletionProgress({required int current, required int target}) =>
-    '$current / $target memos completados';
 
-const progressTotalStudyTime = 'Horas totais de estudos';
+//
+// Progress
+//
+const progressTotalStudyTime = 'Totais de estudos';
```
Comment (ggirotto) [inline lib/application/constants/strings.dart:23]: It shouldn't be `estudo` instead `estudos`? I'm not sure

File: lib/application/constants/strings.dart [hunk 2]
```diff
@@ -16,18 +16,51 @@ const collectionsExploreTab = 'Explorar';
...
+
+const executionYourPerformance = 'Seu desempenho';
+const executionBackToCollections = 'Voltar para as coleções';
+
+const executionWellDone = '## Muito Bem';
+const executionImprovedKnowledgeDescription = 'Você acaba de aprimorar seu conhecimento em:';
 
-String progressDifficultyEmoji(MemoDifficulty difficulty) {
+String executionLinearIndicatorCompletionLabel(String completionDescription) =>
+    'Indicador linear demonstrando que o nível de conclusão desta sessão aprendizado está em $completionDescription';
```
Comment (ggirotto) [inline lib/application/constants/strings.dart:42]: desta sessão **de** aprendizado está em

File: lib/application/coordinator/routes.dart [hunk 3]
```diff
@@ -52,6 +52,33 @@ class ProgressPath extends HomePath {
   String get formattedPath => '/$name';
 }
 
+//
+// Collection Details
+//
+class CollectionDetailsPath extends AppPath {
+  CollectionDetailsPath(this.collectionId);
+  static const name = 'collection_details';
```
Comment (ggirotto) [inline lib/application/coordinator/routes.dart:60]: What do you think about creating a routes file which holds all routes strings?

**Review by ggirotto — CHANGES_REQUESTED** [review]

File: lib/application/constants/strings.dart (hunk 1 above)
Comment (matuella) [reply lib/application/constants/strings.dart:23]: I think both are correct

File: lib/application/coordinator/routes.dart (hunk 3 above)
Comment (matuella) [reply lib/application/coordinator/routes.dart:60]: I don't think it's useful, as they are only used here, and they are already in their own routes

**Review by ggirotto — APPROVED** [review]

**Comment by matuella** [comment]
All updates made and merged in the last PR (#56) in this nested chain of collection execution PRs

**Pull request closed without merging**